daemon/config/config_linux.go

```
@@ -42,15 +42,16 @@ const (
 type BridgeConfig struct {
 	DefaultBridgeConfig
 
-	EnableIPTables           bool   `json:"iptables,omitempty"`
-	EnableIP6Tables          bool   `json:"ip6tables,omitempty"`
-	EnableIPForward          bool   `json:"ip-forward,omitempty"`
-	DisableFilterForwardDrop bool   `json:"ip-forward-no-drop,omitempty"`
-	EnableIPMasq             bool   `json:"ip-masq,omitempty"`
-	EnableUserlandProxy      bool   `json:"userland-proxy,omitempty"`
-	UserlandProxyPath        string `json:"userland-proxy-path,omitempty"`
-	AllowDirectRouting       bool   `json:"allow-direct-routing,omitempty"`
-	BridgeAcceptFwMark       string `json:"bridge-accept-fwmark,omitempty"`
+	EnableIPTables           bool              `json:"iptables,omitempty"`
+	EnableIP6Tables          bool              `json:"ip6tables,omitempty"`
+	EnableIPForward          bool              `json:"ip-forward,omitempty"`
+	DisableFilterForwardDrop bool              `json:"ip-forward-no-drop,omitempty"`
+	EnableIPMasq             bool              `json:"ip-masq,omitempty"`
+	EnableUserlandProxy      bool              `json:"userland-proxy,omitempty"`
+	UserlandProxyPath        string            `json:"userland-proxy-path,omitempty"`
+	AllowDirectRouting       bool              `json:"allow-direct-routing,omitempty"`
+	BridgeAcceptFwMark       string            `json:"bridge-accept-fwmark,omitempty"`
```

Don't think we shipped this one yet; given that this is part of `BridgeConfig`, should we remove the `Bridge` config? We don't do this for any of the other options here, and `BridgeConfig` is part of the name.

---

With this many options specific to "bridge", we should start looking at un-embedding the `BridgeConfig` - at least, I think currently  we embed it, so all these options are at the top-level in `daemon.json`, requiring users to hunt for options related to the bridge.

(Needs some thinking how we can transition from one to the other though!)

---

I'd have used a netlabel for this, but that would have meant inventing a format for its value (csv or whatever). Can do that if necessary, but it seems wrong. The netlabels are hard to use, easy to get wrong - and they're silently ignored when their long and obscure names have a typo.